packages/alchemy/src/react/components/auth/sections/EmailAuth.tsx

```diff
@@ -85,10 +85,12 @@ export const EmailAuth = ({
               iconLeft={<MailIcon />}
               iconRight={
                 hideButton ? (
-                  <ChevronRight
-                    className="match-input cursor-pointer"
-                    onClick={() => form.handleSubmit()}
-                  />
+                  <button
```

Comment:
wrapping this svg with a button since it's clickable for a11y 🫡  (and uses type="submit" for trigger form submission instead of using `form.handleSubmit`)